netsuite.gemspec

```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |gem|
   gem.require_paths = ['lib']
   gem.version       = Netsuite::VERSION
 
-  gem.add_dependency 'savon', '~> 2.3.0'
+  gem.add_dependency 'savon', '>= 2.3.0'
```

Comment:
There is a PR for this, let's leave this out of this PR if possible.

https://github.com/NetSweet/netsuite/pull/172